packages/nextflow-language-server/package.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: nextflow-language-server
+description: A language server for Nextflow.
+homepage: https://github.com/nextflow-io/language-server
+licenses:
+  - MIT
+languages:
+  - Nextflow
+  - Groovy
```

Comment:
```suggestion
```

Does the language server actually support all groovy? I'm not super familiar but that's just one thing that jumps out to me

Reply:
Idk, it should in theory support all Groovy, but it's a DSL in Groovy. So you can drop down and use Groovy. 

Let's just remove it though.